.github/action.sh

```diff
@@ -42,4 +42,9 @@ if [ "${ABICHECK_MERGE:-}" = "auto" ]; then
   [ "$GITHUB_EVENT_NAME" != "pull_request" ] || ABICHECK_MERGE=true
 fi
 
-env "$@" bash "$DIR_THIS/../industrial_ci/src/ci_main.sh"
+if [[ $ICI_SRC_PATH == *run/act/* ]]; then
```

Comment:
I had to relax this test a little bit. Not sure why, but in one of my tests it was `/run/act` instead of `/var/run/act`.